keps/sig-cloud-provider/991-cloud-controller-migration/README.md

```diff
@@ -198,11 +225,11 @@ controllerLeaders:
 ```
 
 Save the leader migration configuration file somewhere, for this example we'll use `/etc/kubernetes/cloud-controller-migration.yaml`.
-Now update the kube-controller-manager to set `--cloud-migration-config /etc/kubernetes/cloud-controller-migration.yaml`.
+Now update the kube-controller-manager to set `--enable-leader-migration --leader-migration-config /etc/kubernetes/cloud-controller-migration.yaml`.
```

Comment:
This is the advanced version. We should probably start with the basic version where you only need '--enable-leader-migration'. However we should explain which components need it in which K8s versions (N -> N+1). The advanced version should also discuss the K8s versions and configs. Especially as the configs change across versions.

Reply:
Updated existing example to use the basic version.

Reply:
About "explain which components need it in which K8s versions" I think this KEP is to *support* leader migration, not creating configuration for current version. We can't be sure if there is another PR moving one of the controller, at least until code freeze. Creating configuration should be trivial and we can just use a plain PR for each release.